lib/ukboot/boot.c

```diff
@@ -111,6 +107,9 @@
 #include <uk/intctlr.h>
 #endif /* CONFIG_LIBUKINTCTLR */
 
+extern char *argv[CONFIG_LIBUKBOOT_MAXNBARGS];
```

Comment:
I feel like these names are a bit "established"? Maybe we should just name them as `boot_argv`/`boot_argc` or something? I am fine with this as well but it's just that `argv` and `argc` sound like something other components may want to unexpectedly define for themselves.

Reply:
Yes,  I thought about that too. But then again these vars really hold `argc` and `argv`. Either way, I don't have a strong opinion.